### PR TITLE
Create PropertyEditor widget

### PR DESCRIPTION
**Pull request by GordonSmith**
Signed-off-by: Gordon Smith <gordon.smith@lexisnexis.com>

### CODE REVIEW

**Comment by GordonSmith** [comment]
@jbrundage. @mlzummo Please Review

File: widgets/src/other/PropertyEditor.js [hunk 1]
```diff
@@ -0,0 +1,355 @@
...
+        "    ;\n" +
+        "}"
+    };
+
+    var formatJSProperties = function (widget, includeColumns, includeData) {
+        if (!widget) {
+            return "";
+        }
+        var propsStr = Persist.discover(widget).map(function (prop) {
+            if (!widget[prop.id + "_modified"]()) {
```
Comment (mlzummo) [inline widgets/src/other/PropertyEditor.js]: getting Uncaught TypeError: undefined is not a function

**Comment by mlzummo** [comment]
JavaScript and Serialization sections blank -- due to the TypeError reported before.

What are the 2 extra blank areas underneath Javascript and Serialization boxs?

**Comment by GordonSmith** [comment]
Resolved - https://rawgit.com/GordonSmith/Visualization/PROPERTY_EDITOR/demos/dermatology.html

**Comment by mlzummo** [comment]
the right hand script box is not working in FF 35.0.1 (os x)

**Comment by GordonSmith** [comment]
FYI it works ok in FF 35.0.1 (linux)...

**Comment by mlzummo** [comment]
http://grab.by/F8N0

**Comment by GordonSmith** [comment]
For sanity can you do a forced clean refresh etc,

**Comment by mlzummo** [comment]
just tried that, also cleared cache, history and restarted FF and still same thing ... very odd

**Comment by dineshshetye** [comment]
I tested the issue where icons were not appearing in the graph widget and this pull request has a fix for it.

File: demos/dermatology.html [hunk 2]
```diff
@@ -102,176 +105,79 @@
...
+
+            function serializeTest(propEditor) {
+                /*
+                document.getElementById("jsText").setAttribute("class", "prettyprint");
+                document.getElementById("jsText").innerText =
+                    "//  JavaScript  ---\n" +
+                    propEditor.getJavaScript("widget", true, true);
+                */
+                document.getElementById("serializationText").setAttribute("class", "prettyprint");
+                document.getElementById("serializationText").innerText =
```
Comment (mlzummo) [inline demos/dermatology.html]: .innerText has support issues on FF Mac use .innerHTML

**Comment by GordonSmith** [comment]
@mlzummo Can you retest with latest?

**Comment by mlzummo** [comment]
latest works

**Pull request merged**